apps/docs/src/app/core/component-docs/tabs/examples/tab-form-element-example/tab-form-element-example.component.ts

```diff
@@ -0,0 +1,8 @@
+import { Component } from '@angular/core';
+
+@Component({
+    selector: 'fd-tab-form-element-example',
+    templateUrl: './tab-form-element-example.component.html',
+    styleUrls: ['./tab-form-element-example.component.scss']
```

Comment:
build is still failing. The .scss file was deleted but is still being referenced here.

Reply:
```
Error: Entry module not found: Error: Can't resolve '/Users/madrov/coding/sap/fundamental-ngx/apps/docs/src/app/core/component-docs/tabs/examples/tab-form-element-example/tab-form-element-example.component.scss' in '/Users/madrov/coding/sap/fundamental-ngx'
```

Reply:
modified.